kitsune/search/es_utils.py

```diff
@@ -134,10 +159,15 @@ def index_object(doc_type_name, obj_id):
         # just return
         return
 
+    kwargs = {}
+    # For ES8, use string "true" instead of boolean True for refresh parameter
+    if settings.TEST:
```

Comment:
You already have this check above

Reply:
There are three checks for `settings.TEST` in the file. The `es_client` has one to modify timeout and retry as part of local testing, `index_object()` and `delete_object()` both test to set the refresh to true. It didn't seem reasonable to pull that check into a helper function just for the second two tests. I don't think we want `refresh` set to true generally, as it forces an immediate refresh which would be expensive in prod.
I think we could remove the code I added for troubleshooting/resolving tests (so the es_client() timeout, retries and retry_on_timeout) and that would get rid of the test in `es_client()`. But we want the test in the other two places for sure.